### DISCUSSION TITLE
Defining Cross-Sections

### DISCUSSION THREAD

**Discussion by jchkoch** [cslotboom/limitstates#10] [General]
Have you considered using external python packages to calculate the load side of the limit states? That way you would not need to reinvent those calculations. [sectionproperties](https://github.com/robbievanleeuwen/section-properties) is a great package for this.

**Comment by cslotboom** [comment]
Hey @jchkoch - definitely. The limitstates analysis library will contain some "hand-calc" type equations which act on elements with simple spans and uniform/point loads, etc.

However, my philosophy is to mostly avoid a lot of analysis within the library, and where appropriate provide "converters" that take limitstate objects and generate the appropriate objects from those libraries.

If there was a particular use case you had in mind I could take a look at it for a future release.
